### ISSUE TITLE
accuracy() for forecast objects

### ISSUE THREAD

**Issue by ellisp** [ellisp/forecastHybrid#23]
Do we want accuracy (compared to a known forecast gold standard) to work on our forecast objects out of the box?  At the moment you need to know to manually specify the mean of the forecast object is what is to be tested.
```R
> series <- M3[[1]]
> x <- series$x      # ie the data to be fitted
> xx <- series$xx    # ie the true, actual values of the forecast period
> h <- length(xx)    # ie the length of the forecast period
> tmp <- forecast(hybridModel(x), h = h)
Warning message:
In hybridModel(x) :
  The stlm model requires that the input data be a seasonal ts object. The stlm model will not be used.
> accuracy(tmp, xx)
Error in NextMethod(.Generic) : cannot assign 'tsp' to zero-length vector
> accuracy(tmp$mean, xx)
               ME     RMSE      MAE      MPE     MAPE     ACF1 Theil's U
Test set 488.2159 635.8854 522.7854 5.853305 6.495892 0.489442 0.7779601
```

**Comment by dashaub** [comment]
Partially fixed here: b5aa4a1f11f893eda8b9faeb3f24ffdf5190c0ee
test set accuracy:
```
w2 <- window(wineind, start = c(1993))
w1 <- window(wineind, end = c(1992, 12))
hm <- hybridModel(w1, "aent")
wfc <- forecast(hm)
accuracy(wfc, w2)
```
train set accuracy also works:
```
accuracy(wfc)
```
I'll implement something similar so that `accuracy` also works on the hybridModel objects.

**Comment by dashaub** [comment]
A hack fixes this for `hybridModel` objects 1f6738f94bb84bdf1df4f33151c6f97246804bed
`accuracy(hybridModel(wineind))` will not work for now with `nnetar`/`stlm` objects. The next release of forecast will fix this issue.

**Comment by dashaub** [comment]
Fixed with release of forecast 7 and here: 1a5ed88fcbe0a2a4dfd352d9f539bef1db976974
